sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates-client-side/README.md

```diff
@@ -0,0 +1,44 @@
+# Azure Spring Boot Key Vault Certificates client library for Java
+
+## Key concepts
+This sample illustrates how to use [Azure Spring Boot Starter Key Vault Certificates ][azure_spring_boot_starter_key_vault_certificates].
+
+This sample should work together with [azure-spring-boot-sample-keyvault-certificates-server-side].
+
+## Getting started
+
+### Prerequisites
+- [Environment checklist][environment_checklist]
+
+### Config the sample
+Fulfill these properties in application.yml:
+```yaml
+azure:
+  keyvault:
+    uri:                 # The URI to the Azure Key Vault used
+    tenant-id:           # The Tenant ID for your Azure Key Vault (needed if you are not using managed identity).
+    client-id:           # The Client ID that has been setup with access to your Azure Key Vault (needed if you are not using managed identity).
+    client-secret:       # The Client Secret that will be used for accessing your Azure Key Vault (needed if you are not using managed identity).
+```
+
+### How to run
+1. Start azure-spring-boot-sample-keyvault-certificates-server-side's SampleApplication
+1. Start azure-spring-boot-sample-keyvault-certificates-client-side SampleApplication
```

Comment:
inconsistent between azure-spring-boot-sample-keyvault-certificates-server-side'**s** and azure-spring-boot-sample-keyvault-certificates-client-side